projects/golang/go/1.21/rpmbuild/SPECS/golang.spec

```diff
@@ -148,8 +148,8 @@ Requires:       %{name}-bin = %{version}-%{release}
 Requires:       %{name}-src = %{version}-%{release}
 Requires:       go-filesystem
 
-Patch1:         0001-Disable-Google-s-proxy-sumdb-and-toolchain.patch
-Patch4:         0004-cmd-link-use-gold-on-ARM-ARM64-only-if-gold-is-avail.patch
+Patch1:         0101-Disable-Google-s-proxy-sumdb-and-toolchain.patch
```

Comment:
Any reason it's 101 even though it's a later version (1.21)?

Reply:
@markapruett These numbers are the [same from fedora](https://src.fedoraproject.org/rpms/golang/tree/rawhide) and we are numbering as utility patches.